test/e2e/multinode/e2e_subscribe_test.go

```diff
@@ -30,6 +30,7 @@ func setUpSubscribe() {
 		SidechainGoLiveTimeout: big.NewInt(0),
 	}
 	setupNewSGNEnv(p)
+	tc.SleepWithLog(10, "sgn syncing")
```

Comment:
why do we need additional sleep now? can it be polling-based?

Reply:
The monitor needs about 10 seconds to start, and It is what we did in all other tests.

Reply:
Did tests pass before adding this 10 seconds sleep?

Reply:
No, the test failed without sleep